### PR TITLE
provide arm64 images for ecotone contributors on mac

### PR DESCRIPTION
**Pull request by jlabedo**

### CODE REVIEW

**Review by dgafka — APPROVED** [review]

File: build.sh [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
-&& docker push simplycodedsoftware/nginx-php:8.0 \
-&& docker build -t simplycodedsoftware/php:8.1 Debian/PHP/8.1/Fpm \
-&& docker push simplycodedsoftware/php:8.1 \
-&& docker build -t simplycodedsoftware/nginx-php:8.1 Debian/PHP/8.1/Fpm-Nginx \
-&& docker push simplycodedsoftware/nginx-php:8.1 \
-&& docker build -t simplycodedsoftware/php:8.2 Debian/PHP/8.2/Fpm \
-&& docker push simplycodedsoftware/php:8.2 \
-&& docker build -t simplycodedsoftware/nginx-php:8.2 Debian/PHP/8.2/Fpm-Nginx \
-&& docker push simplycodedsoftware/nginx-php:8.2
+docker buildx build --platform linux/amd64,linux/arm64 --push --pull -t simplycodedsoftware/debian:latest Debian/Base \
```
Comment (dgafka) [inline build.sh:4]: Can you explain how does it behave without arm64 platform on mac? 
You can't use this docker image or it fails somehow?

**Comment by dgafka** [comment]
Image were rebuilt

File: build.sh (hunk 1 above)
Comment (jlabedo) [reply build.sh:4]: It works well, but it's slow (on M1 cpu mac). Docker for mac is emulating amd64 processor in this case.

Just one thing is the xdebug script that don't select the right host, so I added this env variable in my setup : `XDEBUG_CONFIG="client_host=host.docker.internal"`. I think it is related to Docker for mac

Comment (dgafka) [reply build.sh:4]: Sure, thanks for contribution :)

**Pull request merged**